pkg/oc/admin/prune/builds.go

```diff
@@ -98,11 +98,15 @@ func (o *PruneBuildsOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command,
 	}
 	o.Out = out
 
-	osClient, _, err := f.Clients()
+	config, err := f.BareClientConfig()
```

Comment:
I don't remember this one.  Different than `f.ClientConfig()`?

Reply:
guess the ClientConfig will work just fine here.